src/util/pg.py

```diff
@@ -361,9 +361,19 @@ def alter_column_type(cr, table, column, type, using=None, logger=_logger):
             [model_of_table(cr, table), column],
         )
     if not using:
-        # Simple case. Use general SQL syntax
-        cr.execute('ALTER TABLE "%s" ALTER COLUMN "%s" TYPE %s' % (table, column, type))
-        return
+        # if there is a high number of NULL entries, it will be faster to just ignore those
+        cr.execute(sql.SQL("ANALYZE {}({})").format(sql.Identifier(table), sql.Identifier(column)))
```

Comment:
Ideally we could wait to merge `util.format` and reuse it, since it's less verbose.